### DISCUSSION TITLE
How to deal with CSRF in AJAX requests?

### DISCUSSION THREAD

**Discussion by JohnMaverick** [YetiForceCompany/YetiForceCRM#17492] [How to] [answered]
I have custom module and two files client-side and server-side. Client side designing custom UI and server-side is responsible for saving record. As Yeti by default including CSRF protection it needs some dealing with tokens etc. Can developer get some guidance on that?
It giving me constantly 400 (Bad Request) on save

Cheers!

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
use AppConnector
```js
AppConnector.request({
	module: 'Home',
	action: 'BrowsingHistory'
}).done(function (response) {
	
});
```
